dashboards/aws-lex/aws-lex.json

```diff
@@ -0,0 +1,297 @@
+              "zero": true
+            }
+          }
+        },
+        {
+          "title": "Latency",
+          "layout": {
+            "column": 6,
+            "row": 4,
+            "width": 7,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.area"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountIds": [],
+                "query": "FROM Metric select average(aws.lex.RuntimeSucessfulRequestLatency) as RuntimeSuccessfulRequestLatency facet aws.lex.BotAliasId TIMESERIES "
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        },
+        {
+          "title": "Runtime Errors",
+          "layout": {
+            "column": 1,
+            "row": 7,
+            "width": 4,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.stacked-bar"
+          },
+          "rawConfiguration": {
+            "colors": {
+              "seriesOverrides": [
+                {
+                  "color": "#e70814",
+                  "seriesName": "RuntimeUserErrors"
+                }
+              ]
+            },
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountIds": [],
+                "query": "FROM Metric select count(aws.lex.RuntimeUserErrors) as RuntimeUserErrors, count(aws.lex.RuntimeSystemErrors) as RuntimeSystemErrors timeseries"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        },
+        {
+          "title": "Bot Channel Errors",
+          "layout": {
+            "column": 5,
+            "row": 7,
+            "width": 4,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.line"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountIds": [],
+                "query": "FROM Metric SELECT count(aws.lex.BotChannelResponseCardErrors) as BotChannelResponseCardErrors, count(aws.lex.BotChannelSystemErrors) as BotChannelSystemErrors, count(aws.lex.BotChannelAuthErrors) as BotChannelAuthErrors, count(aws.lex.BotChannelConfigurationErrors) as BotChannelConfigurationErrors TIMESERIES since 3 weeks ago"
```

Comment:
Why is this for three weeks but no other widgets are for that time range? / If we want to include this for 3 weeks, the title of this widget should reflect that saying `Bot channel errors since 3 weeks ago`

Reply:
addressed